### PR TITLE
[RDAPI-27966] Added 7.7 August 2022 'com.axway.apigw.cookie.validation.ignore' system property.

### PR DESCRIPTION
**Pull request by moscaribera**
## Describe the changes

Added new *7.7 August 2022* section to _System properties changes_ page.

## Deploy preview link

[Please add the deploy preview link to the **specific page** that you've changed.

https://deploy-preview-2384--axway-open-docs.netlify.app/docs/apim_reference/system_props/#77-august-2022

## Checklist for contributors

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

### CODE REVIEW

File: content/en/docs/apim_reference/system_props.md [hunk 1]
```diff
@@ -470,3 +470,11 @@ Axway-defined Java system properties introduced in the 7.7 20220530 release
...
 |com.axway.apimanager.securitydevice.httpheaders.propagate | API Manager | When set to 'true' the headers generated by the API Manager Inbound Security Invoke Policy Security device are propagated in http.headers for further API Manager processing overriding headers received in the request.  Default value: 'false' |
 |com.axway.apimanager.securitydevice.authz.legacy.enabled | API Manager | When set to 'true' allows you to store external client ids, API keys, or OAuth tokens in any of the Security Device token stores in API Manager Applications. Default value: ‘false' |
+
+## 7.7 August 2022
+
+Axway-defined Java system properties introduced in the 7.7 20220830 release
+
+| System Property                                  | Context     | Description   |
+| ------------------------------------------------ | ----------- | ------------- |
+| com.axway.apigw.cookie.validation.ignore | API Gateway | Allow HTTP Cookie related filters to do not enforce cookie validation as per RFC 6265. When set to 'true' the API Gateway handles HTTP cookies as in 7.7 March 2021 release. Default value: 'false' |
```
Comment (cpalmeraxway) [inline content/en/docs/apim_reference/system_props.md:480]: Remove "do" so to have "to not enforce"

**Review by cpalmeraxway — APPROVED** [review]

File: content/en/docs/apim_reference/system_props.md (hunk 1 above)
Comment (andreamussap) [reply content/en/docs/apim_reference/system_props.md:480]: @moscaribera @cpalmeraxway "API Gateway handles HTTP cookies **as in 7.7 March 2021 release**" -> This is too generic. What does "as in 7.7 March 2021 release" means?

Comment (moscaribera) [reply content/en/docs/apim_reference/system_props.md:480]: Customers complaint the behavior changed after a system update, so the flag restores the old behavior.

Comment (andreamussap) [reply content/en/docs/apim_reference/system_props.md:480]: (_Customers complaint the behavior changed_) --> Ah, ok. But, remember that not everybody who reads this doc is a customer already, and not ALL customers complained about that, so this information "as in 7.7 March 2021 release" is irrelevant, and in fairness, confusing, to many people reading this doc. Moreover, the most important information is there, when you say that "not enforce cookie validation as per RFC 6265". So can you please update it as per Ciaran suggestion, and remove the irrelevant info, so I can merge the PR. The final version will be something like ??

> When set to 'true', allow HTTP Cookie related filters to not enforce cookie validation as per RFC 6265. Default value: 'false'

Comment (moscaribera) [reply content/en/docs/apim_reference/system_props.md]: Done.

**Pull request merged**